docs/source/markdown/podman-container-checkpoint.1.md

```diff
@@ -28,6 +28,58 @@ archives. Not compressing the checkpoint archive can result in faster checkpoint
 archive creation.\
 The default is **zstd**.
 
+#### **--create-image**=*image*
+
+Create a checkpoint image. This is an OCI image with single layer (tar file)
+that contains all checkpoint files. The content of the image is in the same
+format as tar.gz file created with **--export**. The checkpoint image can be
+pushed to a standard container registry and pulled on a different system. It can
+also be exported locally with **podman image save** and inspected with **podman
+inspect**. Inspecting a checkpoint image would display additional information,
+stored as annotations, about the host environment used to the checkpoint:
```

Comment:
You make several mentions of "container" below in the explanation of the various fields, but the work "container" doesn't appear once in the description, just image.  Maybe the first line should be "Create a checkpoint image from the provided container." or some such.

Reply:
Thanks, I've updated this paragraph.